scripts/bundle_clusterfuzz.py

```diff
@@ -97,6 +100,14 @@
     binaryen_bin = shared.options.binaryen_bin
     binaryen_lib = shared.options.binaryen_lib
 
+# ClusterFuzz's run.py uses these features. Keep this in sync with that, so that
+# we only bundle initial content that makes sense for it.
+features = [
+    '-all',
+    '--disable-shared-everything',
+    '--disable-fp16',
+]
```

Comment:
Would it be feasible to deduplicate these into a shared location?

Reply:
Not trivially, since `run.py` is not part of the normal "tree". We never run it as part of our own code, only after being packaged for ClusterFuzz. So we don't have the normal shared locations. I suppose we could have a file that we import locally and copy over for ClusterFuzz, but given this is just a few lines, that seems a bit excessive to me.